### ISSUE TITLE
[ransomware.live] improvements

### ISSUE THREAD

**Issue by yassine-ouaamou** [OpenCTI-Platform/connectors#2665]
Following some tests after the improvements made by @sudesh0sudesh in this issue https://github.com/OpenCTI-Platform/connectors/issues/2351 , here are two other improvements I see:
- Use "related-to" relationship between an Individual and a Sector 
     I have seen the following error in the ingestion: `The relationship type part-of is not allowed between Individual and Sector`
- Remove the victim's webpage URL from the external references (in reports for example)

**Comment by sudesh0sudesh** [comment]
@yassine-ouaamou sorry but i am creating it between organisation and sector

**Comment by sudesh0sudesh** [comment]
sorry, there is one place i used individual, will modify it 👍

**Comment by yassine-ouaamou** [comment]
Thanks @sudesh0sudesh!
Why would you need to link an Individual with a Sector? Can you share with us an example, please?

**Comment by sudesh0sudesh** [comment]
Nope, it was a mistake, I was testing as organisations with name less than two words, I should have replaced it in organisation.

**Comment by yassine-ouaamou** [comment]
I noticed that the connector ingests the full data each time:
![image](https://github.com/user-attachments/assets/1c40ab1d-a46c-43d4-8e56-af166baca7b3)
![image](https://github.com/user-attachments/assets/54be6d2a-fce0-4cd0-b505-c822dae96d26)
This is a blocking behaviour as it will impact the performance of the platform.
Is it possible to implement an offset in order to fetch only the new data?

**Comment by sudesh0sudesh** [comment]
On the other hadn, they can increase the frequency of ingestion

**Comment by yassine-ouaamou** [comment]
What could be the issues with the reports in the case you are describing?

**Comment by sudesh0sudesh** [comment]
Few of those are Wrong Country assignment, assignments to Wrong org,

**Comment by seanthegeek** [comment]
I'm also observed where the  victim  is linked to a part of Diplomacy when the sector field in ransomeware.live is blank and the victim has nothing to do with Diplomacy. 

![image](https://github.com/user-attachments/assets/d9a88fcb-d028-41dd-92d1-620771c8413f)

![image](https://github.com/user-attachments/assets/a093c6fc-d341-4d2e-ad8e-0e90a6789552)

**Comment by seanthegeek** [comment]
It would be great to be able to turn of the generation of threat actors. I'm using intrusion sets exclusively instead of threat actors to keep things simple.

**Comment by sudesh0sudesh** [comment]
@seanthegeek will be looking at both of those, will priortise sector and will be making threat actors optional in future release

**Comment by seanthegeek** [comment]
@sudesh0sudesh Thanks. I just thought of other improvements for future releases:

- Add relationships between the intrusion sets and the tools used (Most of the tools exist in the MITRE dataset)
- Add the links provided in the ransomware.live group pages as external references in the intrusion sets
- Add a link to ransomeware.live group page to the external references of the intrusion set

**Comment by seanthegeek** [comment]
The ransomware.live does not currently provide the list of tools or YARA rules via the API. I'll contact them about that. The reference links are included in a list named `profile` though.

**Comment by fwuest** [comment]
I am currently testing a modified connector which sanitizes victims that have an "*" or a "?" in the title. I have also modified the activity and sector code to sanitize blanks, spaces and Not Found. Older data is no longer ingested with the wrong sector Diplomacy but with no sector as this information is not available. See issue: [2841](https://github.com/OpenCTI-Platform/connectors/issues/2841)
